projects/netgrif-components/src/lib/data-fields/number-field/number-currency-field/number-currency-field.component.html

```diff
@@ -11,4 +11,4 @@
     <mat-hint>{{numberField.description}}</mat-hint>
     <mat-error *ngIf="numberField.isInvalid(formControlRef)">{{getErrorMessage()}}</mat-error>
 </mat-form-field>
-<mat-label class="currency-label" *ngIf="fieldType == 'number'">{{getCurrencySymbol()}}</mat-label>
+<mat-label class="currency-label" *ngIf="fieldType === 'number'">{{getCurrencySymbol()}}</mat-label>
```

Comment:
the check could be done in the class and exposed to the html as a method. Then the 'number' constant could be a constant and not a static string

Reply:
Implemented.